packages/react-component-library/src/components/index.ts

```diff
@@ -1,3 +1,4 @@
+// Components
 import Avatar from './Avatar'
```

Comment:
One thing I want to change to. Don't have a default export in a component and then in the index for components use:

export * from './TextInput'

Reply:
Sounds good to me.